content/800-data-platform/06-query-console.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Query Console'
+metaTitle: 'Query Console'
+metaDescription: 'Query Console'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Use the Query Console to run the 3 pre-defined `findMany()` test queries or add and test queries of your own using the [Prisma Client API for model queries](/reference/api-reference/prisma-client-reference#model-queries).
+
+</TopBlock>
+
+## UI components
+
+The Query Console is comprised of 2 major UI components - **Query Editor** and **Query Results Console**.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657148399/docs/pdp/pdp-query-console-components_w98dp5.png" alt="Query Console - UI Components"/>
+
+| UI component          | Description                                                                         |
+| --------------------- | ----------------------------------------------------------------------------------- |
+| Query Editor          | • Add, edit, delete queries using the Prisma Client API<br />• Run a selected query |
+| Query Results Console | View the results from the selected query                                            |
+
+## Default model queries
+
+- `findMany()`
+- `findMany()` in which the `post_id` is 0
+- `findMany()` but return the data from only 2 fields
```

Comment:
Here are what I think are the default queries with their current comment:

```
// Find the first 100 rows in the `User` table
await prisma.user.findMany({ take: 100 })

// Find the first 100 rows in the `User` table that match a criteria
await prisma.user.findMany({
  where: {
    "id": "example"
  },
  take: 100 
})

// Find the first 100 rows in the `User` table, but only return a few columns
await prisma.user.findMany({
  select: {
    "id": true,
    "createdAt": true
  },
  take: 100
})
```

Reply:
I've replaced these with excerpts from the code sample. Thanks for pasting it here!